my-app/src/Components/UI/OpenDialog.tsx

```diff
@@ -28,7 +28,7 @@ interface PropsDialogModal {
           {'Создать задачу?'}
```

Comment:
for the name of folders. Use lowercase everywhere: Components => components, UI => ui